be/src/storage/column_in_predicate.cpp

```diff
@@ -365,6 +367,93 @@ class BinaryColumnInPredicate : public ColumnPredicate {
     ItemHashSet<Slice> _slices;
 };
 
+class DictionaryCodeInPredicate : public ColumnPredicate {
+private:
+    enum LogicOp { ASSIGN, AND, OR };
+
+public:
+    DictionaryCodeInPredicate(const TypeInfoPtr& type_info, ColumnId id, const std::vector<int32_t>& operands,
+                              size_t size)
+            : ColumnPredicate(type_info, id), _bit_mask(size) {
+        for (auto item : operands) {
+            DCHECK(item < size);
+            _bit_mask[item] = 1;
+        }
+    }
+
+    ~DictionaryCodeInPredicate() override = default;
+
+    template <LogicOp Op>
+    inline void t_evaluate(const Column* column, uint8_t* sel, uint16_t from, uint16_t to) const {
+        const Int32Column* dict_code_column;
+        if (column->is_nullable()) {
```

Comment:
Use `ColumnHelper::get_data_column()` instead?

Reply:
Got it. will do it.